pkg/cluster/cluster.go

```diff
@@ -108,7 +108,7 @@ type Options struct {
 	// If not set this will create the default DelegatingClient that will
 	// use the cache for reads and the client for writes.
 	// NOTE: The default client will not cache Unstructured.
-	NewClient NewClientFunc
+	NewClient client.NewClientFunc
```

Comment:
The comment that says `DelegatingClient` should be updated.

Reply:
fixed, thanks!